Rakefile

```diff
@@ -1,11 +1,8 @@
-require 'govuk_tech_docs'
-require_relative './app/requires'
+require "govuk_tech_docs"
+require "rspec/core/rake_task"
+require_relative "./app/requires"
 
-begin
-  require 'rspec/core/rake_task'
-  RSpec::Core::RakeTask.new(:spec)
-rescue LoadError
-end
+RSpec::Core::RakeTask.new(:spec)
```

Comment:
This shouldn't fall into the same [problem as we encountered in the cache-clearing-service](https://github.com/alphagov/cache-clearing-service/pull/260), as RSpec isn't listed as a _developer_ dependency in govuk-developer-docs. It probably should be, but that's out of scope for this PR!